internal/controllers/nmc_reconciler.go

```diff
@@ -508,8 +514,9 @@ const (
 	configFileName = "config.yaml"
 	configFullPath = volMountPoingConfig + "/" + configFileName
 
-	volNameConfig       = "config"
-	volMountPoingConfig = "/etc/kmm-worker"
+	volNameConfig          = "config"
+	volNameImageRepoSecret = "image-repo-secret"
```

Comment:
Where is this name is set into the VolumeMount?

Reply:
Now correctly used [here](https://github.com/kubernetes-sigs/kernel-module-management/pull/573/files#diff-4abf41a7395eeba9459df6b2fe59c88e93e33e83dd683d40878c61651de912f6R900). Thanks!